cf/scf-develop.yaml.erb

```diff
@@ -266,6 +275,27 @@ jobs:
         params:
           file: out/version
 
+  - name: package-bits
+    plan:
+      - aggregate:
+        - get: src
+          passed:
+          <% if manual_pull_sources %>
+          - pull-sources
+          <% end %>
+          trigger: true
+        - get: ci
+        - get: s3.certstrap-binary
```

Comment:
Note, I dropped darwin because I only got linux from the certstrap resource. Will check what is going with that too.